sdk/core/Azure.Core.Experimental/src/Primitives/BinaryData.cs

```diff
@@ -188,7 +190,7 @@ public static BinaryData Create(string data, Encoding encoding)
         /// Converts the BinaryData to a stream.
         /// </summary>
```

Comment:
Should we consider just exposing the `Data` property directly, since there's no underlying conversion to be done.   Seems a bit odd that we'd have an accessor function rather than using a getter.   In the case of other formats, the `ToXXX` makes sense to me because it signifies that there's an underlying conversion needed and that work needs to happen.  With the name changes, there doesn't seem to be a need for an `AsBytes` to keep a consistent pattern of naming anymore.

Reply:
There is still an implicit conversion to ReadOnlyMemory. I guess AsBytes gives us more flexibility for changing underlying Data type in the future.